spec/models/remote_translation_spec.rb

```diff
@@ -49,6 +49,14 @@
     expect(remote_translation).to be_valid
   end
 
+  it "checks app locales that need to be mapping" do
```

Comment:
There seems to be a typo here. Maybe something like:

```suggestion
  it "is valid with a locale that uses a different name in the remote service" do
```